### DISCUSSION TITLE
Collection Icon problem

### DISCUSSION THREAD

**Discussion by wuchunhuan** [outline/outline#7573] [General]
I noticed a lock was added to one of my collection icon at the bottom right corner in the Collections list, other collections are not affected though. 
![image](https://github.com/user-attachments/assets/2f577fe4-6522-4708-af46-5acdd268f12a)

First I thought it may have something to do with the collection permission, but i never changed it or any other settings, and I have another collection with the same permission(all members: No access, me: Manage), its icon has no lock attached to it.

 I tried to change to other icon, and no lock is attached, and then change back to the original one, lock is back on(the original icon has no lock), so wierd... 

what bothers me is that I dont't know what triggers this icon problem, I think it's bug, the only change I made was upgrading to the latest version: v0.79.1, any idea how to fix this?

**Comment by tommoor** [comment]
The lock represents that this collection has restricted access, it was changed in `0.79.1`, as you mention. You can still change it to any other icon.
